benchmarking.py

```diff
@@ -0,0 +1,56 @@
+import matplotlib.pyplot as plt
+import timeit
+import csv
+
+class Graph:
+    #static 2d array to store data
+    taskData = [[],[],[],[],[]]
+
+    #initialize function to start our graph
+    def __init__(self):
```

Comment:
Where is this code actually being ran? In server.py? Do we need a constructor there and need to be calling things like addPoint in certain places?

Reply:
During testing I was running this in server.py, but I was unsure where we wanted to pull in these values. We would need to create the constructor there and call these functions -- I took it out after doing time testing but we would just place it in where appropriate.